src/ZPublisher/tests/test_utils.py

```diff
@@ -33,5 +35,8 @@ def test_unicode(self):
         self.assertEqual(self._makeOne(u'foö'), u'foö')
 
     def test_utf_8(self):
-        self.assertEqual(self._makeOne('test\xc2\xae'), u'test\xae')
+        if PY2:
+            self.assertEqual(self._makeOne('test\xc2\xae'), u'test\xae')
+        else:
+            self.assertEqual(self._makeOne('test\xc2\xae'), u'test\xc2\xae')
```

Comment:
I'm not sure how else to write this test. The u'' -> u'' test is right above this and the b'' -> u'' test below. This wants to test the conversion of a prefix-less string, which means something different in Py 2/3.